portfolio.js

```diff
@@ -267,4 +267,36 @@ form.addEventListener('submit', (e) => {
     formButton.reportValidity();
     e.preventDefault();
   }
-});
\ No newline at end of file
+});
+
+// set up local storage
+
+const nameInput = document.getElementById('name');
+const mailInput = document.getElementById('email');
+const messageInput = document.getElementById('message');
+
+const nameValue = nameInput.value;
+const mailValue = mailInput.value;
+const messageValue = messageInput.value;
```

Comment:
- [ ] The form data for all inputs should be stored in a single object, and saved to a single local storage registry, please consider modifying this part of the code so your code meets the requirements
e.g: 
```
const formData = {
    name: nameInput.value,
    email: emailInput.value,
    body: bodyInput.value,
};
```